addons/website/tests/test_ui.py

```diff
@@ -154,6 +154,12 @@ def media_dialog_undraw(self):
         self.start_tour("/", 'website_media_dialog_undraw', login='admin')
 
 
+@odoo.tests.tagged('external', '-at_install', 'post_install')
```

Comment:
As discussed in PM, you can/should probably keep this test in the regular testing suite and remove the call to undraw by mocking the routes used to fetch images.
Maybe something like `test_02_image_upload_progress_unsplash` but I know there is some mock in js too somewhere, maye more in QUnit tho.

I don't think actually care about the call to undraw itself, you just care about what the media dialog business code will do with the result, right? Mocking the call to return fake results is then the way to go to keep the tour in the regular testing suite instead of having to move it in the nightly only (due to `external` tag)

Reply:
Following our discussion:
Mocking the calls to the Undraw API implies mocking both `media_library_search()` (when searching for images in the media dialog) and `save_library_media()` (after clicking, when saving the selected media as an attachment).
This would only test the JS part (the mutex), but not the fix in `save_library_media()` as it would be mocked.
Given that, we'll keep the test external. (I'll explain this in the commit message for the record.)